README.rst

```diff
@@ -1,13 +1,13 @@
 SM64 Randomized Asset Generator
 ===============================
 
-Generates non-copyrighted randomized assets so sm64 and sm64-port can be used
```

Comment:
This original line is correct. I wanted to reference the names of the repo. We can put them in backticks to codify them if you think that looks better.